test/unittests/skills/core.py

```diff
@@ -106,17 +107,12 @@ def test_load_regex_from_file_none(self):
         self.check_regex_from_file('invalid/none.rx')
 
     def test_load_regex_from_file_invalid(self):
-        try:
```

Comment:
This seems like a mix of stylistic/cleanup changes, not necessarily something that's required to cut ties with py2.7

Reply:
Long time since I went though these but I believe there was some issue with assertEquals matching the exception string (the string had changed) so I changed it to check that a re.error was raised.